docs/03-tutorials/00-service-management/smgt-01-deploy-service.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Deploy an SAP BTP service in your Kyma cluster
+---
+
+This tutorial describes how you can deploy a simple SAP BTP audit log service in your Kyma cluster using the SAP BTP service operator.
+
+## Prerequisites
+
+- [Kyma cluster](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/02-install-kyma/) running on Kubernetes version 1.19 or higher
+- SAP BTP [Global Account](https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/d61c2819034b48e68145c45c36acba6e.html?locale=en-US) and [Subaccount](https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/55d0b6d8b96846b8ae93b85194df0944.html?locale=en-US)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) v1.17 or higher
+- [helm](https://helm.sh/) v3.0 or higher
+- [jq](https://stedolan.github.io/jq/download/)
+<!-->Should we list it here?<--->
+
+## Steps
+
+>**NOTE:** For the BTP service operator to work, you must [disable Istio sidecar proxy injection](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/smsh-01-istio-disable-sidecar-injection#documentation-content) which is enabled on the Kyma clusters by default. The commands in this tutorial already mark the created Namespaces with the **istio-injection=disabled** flag, so you don't have to do this manually.  
+
+1. Create a Namespace and install cert-manager on it. Cert-manager is an obligatory tool that will manage certificates in your cluster. Run:
```

Comment:
"Cert-manager is an obligatory tool that will manage certificates in your cluster" -> change to: "cert manager is required for btp-operator. Omit that step if you have cert-manager already installed"